### PR TITLE
Features and Bug Fixes for Dashboard

### PR DESCRIPTION
**Pull request by brene**
- migration values and default values for lists
- Getting Started doesn't get blocked by the bubble
- Infinite loop after renaming project fixed
- project settings overlay closes after resetting project
- and the stuff that @schickling did
- a new color scheme
- fixed a bug, where the view is reloaded before all items are deleted

### CODE REVIEW

File: src/styles/variables.scss [hunk 1]
```diff
@@ -12,10 +12,10 @@ $grey1: #FBFBFB;
...
 $grey3: #B9B9C8;
 $grey4: #70738C;
-$grey5: #2B3236;
+$grey5: #171A1F;
 $grey6: #192121;
 
 $headerHeight: 66px;
 
-$newGrey3: #8E96A3;
+$newGrey3: #1B1F25;
```
Comment (schickling) [inline src/styles/variables.scss]: What's the reason for this change of colors?

Comment (brene) [inline src/styles/variables.scss]: New color scheme from @julianbauer is implemented

Comment (sorenbs) [inline src/styles/variables.scss]: New colors are so much better
